src/modules/ROOT/pages/changelog/202209/02.json

```diff
@@ -0,0 +1,393 @@
+        "Now they are using separate timestamp."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-server/commit/1fa5f39ab2132b196b543bb8b4f376abca3bc24c",
+        "https://github.com/cortezaproject/corteza-webapp-admin/commit/c63df822a2a276bdc29bdf792c7527fa74e1f9de"
+      ]
+    },
+    {
+      "what": [
+        "Exporting a NS where the name used characters like /, . broke the export because they broke the URL.",
+        "Same for record export but with module name."
+      ],
+      "how": [
+        "URI encoding the filename and replacing dots with -, which caused the URL to be broken"
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-webapp-compose/commit/1f0ea56cf1348fc60a51f489e61afe69d51ca736",
+        "https://github.com/cortezaproject/corteza-webapp-compose/commit/920cc5a0d62f47272a15090cfe1f4b35de3efca1"
+      ]
+    },
+    {
+      "what": [
+        "JSON codecs getting set to the field name instead of values, when editing a module/configuring field mapping."
+      ],
+      "how": [
+        "They are now correctly defaulted."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-webapp-compose/commit/580db32585ecc9ae348314e08279c8f7c52f3ec6"
+      ]
+    },
+    {
+      "what": [
+        "DAL connection configuration displaying the default DSN connection parameters instead of the configured ones."
+      ],
+      "how": [
+        "By properly merging the default values and user-provided configurations, as well as fixing edge-cases where the configuration was made available after the page had loaded.",
+        "In the later case, the user-provided configuration was never displayed instead of the default values."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-webapp-admin/commit/0b41074d9e9f8c04d50864a1abeb069ce92fc836",
+        "https://github.com/cortezaproject/corteza-js/commit/f9f7e920f85cbf2b554640c86d507917cc8da375"
+      ]
+    },
+    {
+      "what": [
+        "Returning value getting corrupted in JavaScript due to unsupported uint64"
+      ],
+      "how": [
+        "Replaced []uint64 with []string for filter.workflowID that corrupted the input value for JavaScript (uint64 vs int64).",
+        "Also implements actual filtering by workflow on RDBMS that was missing"
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-server/commit/ca7d4b00c01c648b7cf0a2ac90c2afb2620a9749"
+      ]
+    },
+    {
+      "what": [
+        "Record reference check"
+      ],
+      "how": [
+        "By using module-id from module field options and not field's module ID"
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-server/commit/a8d2fef3cc7552461580439742b76e337c7744a2"
+      ]
+    },
+    {
+      "what": [
+        "Broken sub-workflow filtering"
```

Comment:
Why was it broken? How did it behave?

`Fixed ...what was the unexpected behaviour users would be expecting...` -- `Fixed improper sub-workflow filtering where only sub-workflows created on a full moon were created` (for example)